react/src/pages/EndpointDetailPage.tsx

```diff
@@ -298,7 +296,7 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
     },
     {
       label: t('modelService.DesiredSessionCount'),
-      children: endpoint?.desired_session_count,
+      children: endpoint?.desired_session_count ?? endpoint?.replicas,
```

Comment:
Is there any reason to consider desired_session_count first? In `ServiceLauncherPageContent`, it uses replicas value first like `replicas: endpoint?.replicas ?? endpoint?.desired_session_count ?? 1,`

same as below.